Plugins/Name.cs

```diff
@@ -45,7 +42,7 @@ public Name(string value)
 
 			var parts = value.Split('/');
 
-			if (parts.Length != 2) throw new ArgumentException("Invalid plugin name format, plugin name must be in \"vendor/project\" format.", nameof(value));
+			if (parts.Length != 2) throw new ArgumentException($"Invalid plugin name format {value}, plugin name must be in \"vendor/project\" format.", nameof(value));
```

Comment:
If `nameof(value)` doesn't display in the error message then `{value}` is good but should be wrapped in quotes, `\"{value}\"`

Reply:
Added quotes